app/frontend/src/components/RatingsSlider/RatingsSlider.tsx

```diff
@@ -88,16 +96,17 @@ export default function RatingsSlider({ defaultValue, onChange }: SliderProps) {
         mark: classes.mark,
       }}
       aria-label={RATINGS_SLIDER}
-      defaultValue={defaultValue}
+      value={currentValue}
       min={0}
       max={1}
       step={0.01}
       marks={marks}
       valueLabelDisplay="on"
       valueLabelFormat={(value) => <SliderLabel value={value} />}
       onChange={(event, value) => {
-        handleSliderChange(sliderRef, value);
-        onChange?.(value);
+        typeof value === "number" && setCurrentValue(value);
+        handleSliderChange(sliderRef, currentValue);
```

Comment:
This should no longer be necessary - the styles will dynamically update from the colour you pass into `useStyles`

Reply:
I was just waiting on [this](https://github.com/Couchers-org/couchers/pull/1264#issuecomment-850826462)

Reply:
Don't worry about the dark reader - we can't account for other people's code

Reply:
Nice, I just wanted to be 100% sure, thanks!